glib-macros/src/properties.rs

```diff
@@ -723,10 +716,10 @@ pub fn impl_derive_props(input: PropsMacroInput) -> TokenStream {
                 item
             });
         quote! {
-            #trait_def
-            impl #trait_ident for #wrapper_type {
-                #(#impls)*
+            pub trait #trait_ident: #crate_ident::IsA<#wrapper_type> {
```

Comment:
Can we seal the traits similar to how we do for Ext/ExtManual traits?

Reply:
Considering #1205, we don't have to